backend/src/apiserver/template/v2_template.go

```diff
@@ -217,3 +271,17 @@ func (t *V2Spec) RunWorkflow(modelRun *model.Run, options RunWorkflowOptions) (u
 	executionSpec.SetPodMetadataLabels(util.LabelKeyWorkflowRunId, options.RunId)
 	return executionSpec, nil
 }
+
+func isPlatformSpec(template []byte) bool {
+	var platformSpec pipelinespec.PlatformSpec
+	templateJson, err := yaml.YAMLToJSON(template)
+	if err != nil {
+		return false
+	}
+	err = protojson.Unmarshal(templateJson, &platformSpec)
+	if err != nil {
+		return false
+	}
+	_, ok := platformSpec.Platforms["kubernetes"]
```

Comment:
The function name should match what it checks, per this line of code, only `kubernetes` platform spec should return `True`, if that's the intention, the function name should reflect `kubernetes` as well.

Reply:
Thank you! I just updated the function name.